lms/djangoapps/course_goals/templates/course_goals/edx_ace/goalreminder/email/body.html

```diff
@@ -66,7 +53,12 @@
 
                 <a style="color: white; text-decoration: none; display: inline-block;"
                     href="{{course_url}}" target="_blank">
-                    <div style="padding: 0.5rem 0.75rem; background: #D23228; margin-bottom: 1rem;">
+                    <div style="
+                        padding: 8px 0 12px;
```

Comment:
outlook (and potentially other email clients) don't support rem, so need px as a fallback